articles/article-063-reverse-engineering.md

```diff
@@ -0,0 +1,26 @@
+<!-- BEGIN TITLE -->
+# Reverse Engineering
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+In general terms, *Reverse Engineering* describes the process of analyzing an existing apparatus to figure out how it works and how it was made.
```

Comment:
Is "In general terms" really needed? The sentence seems more clear to me without it.